packages/components/src/components/CodeSnippet/CodeSnippet.stories.tsx

```diff
@@ -16,7 +16,7 @@
 
 import React from 'react';
 import { CodeSnippet } from './CodeSnippet';
-import { InfoCard } from '../../layout/InfoCard';
+import { InfoCard } from '@backstage/core';
```

Comment:
what is it that made the card stay in core?

Reply:
No specific reason. I've just mechanically moved "components" folder for the sake of checking if it works. I'm going to fix the leftovers and do a proper (not draft) PR.